### PR TITLE
Prevent DMPU overlowing crossword container

### PR DESCRIPTION
**Pull request by adamnfish**
Layout bug fix!

I recommend viewing the change with the `w=1` flag to [ignore whitespace changes](https://github.com/guardian/frontend/pull/20507/files?w=1) (indentation).

## What does this change?

Quick crosswords are smaller and have fewer clues. In some cases the height of the crossword / clues is less than the hight of a DoubleMPU ad, which led to the ad overflowing its container and running over the comment ad, beneath. This PR fixes that layout issue by adding a minimum height to the crossword container at the widest size where ads are displayed. At other breakpoints we leave the layout as-is.

This also improves the layout for ad-free. The RH-space for the ad only appears if ads are enabled.

### With ad, properly spaced.
crossword-ad-fix

### Without ad, no empty space at RHside.
crossword-ad-fix-ad-free

## What is the value of this and can you measure success?

Better crosswords layout!

### Does this change break ad-free?

It improves ad-free!

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by jeteve — APPROVED** [review]

File: applications/app/views/fragments/crosswords/crosswordContent.scala.html [hunk 1]
```diff
@@ -5,46 +5,49 @@
...
+        <article id="crossword" class="content content--article tonal tonal--tone-news" role="main">
+            @crosswordMetaHeader(crosswordPage)
 
-    <div class="content__main tonal__main tonal__main--tone-news">
-        <div class="gs-container">
-            <div class="js-content-main-column">
+            <div class="content__main tonal__main tonal__main--tone-news @if(adsEnabled) {crossword--with-ad} else {}">
+                <div class="gs-container">
+                    <div class="js-content-main-column">
+                        <div class="@if(adsEnabled) {crossword__spacer--ad}">
```
Comment (katebee) [inline applications/app/views/fragments/crosswords/crosswordContent.scala.html]: Random Play template question:

Line 13 uses: `@if(adsEnabled) {crossword--with-ad} else {}` ...why does this need the `else {}` while it seems okay to omit it here ? `@if(adsEnabled) {crossword__spacer--ad}`

**Review by katebee — APPROVED** [review]
👍 🎉

File: applications/app/views/fragments/crosswords/crosswordContent.scala.html (hunk 1 above)
Comment (adamnfish) [reply applications/app/views/fragments/crosswords/crosswordContent.scala.html]: It doesn't, you're right! Inconsistent tidying up. Great spot, thanks.

Comment (adamnfish) [reply applications/app/views/fragments/crosswords/crosswordContent.scala.html]: In fact, crossword-with-ads wasn't supposed to be there at all (intermediate testing) so doubly well spotted, thanks again.

**Pull request merged**